daisy_workflows/image_build/windows/post_install.ps1

```diff
@@ -359,6 +359,43 @@ function Set-Repos {
   Run-Command 'C:\ProgramData\GooGet\googet.exe' -root 'C:\ProgramData\GooGet' addrepo 'google-compute-engine-stable' 'https://packages.cloud.google.com/yuck/repos/google-compute-engine-stable'
 }
 
+function Export-ImageMetadata {
+  $version = (Get-ComputerInfo).OsVersion
+  $family = 'windows-' + (Get-ComputerInfo).windowsversion
+  $name =  (Get-ComputerInfo).OSName
+  $release_date = (Get-Date).ToUniversalTime()
+  $image_metadata = @{'family' = $family;
+                      'version' = $edition;
+                      'name' = $name;
+                      'location' = ${script:outs_dir};
+                      'build_date' = $release_date;
+                      'build_repo' = $stage;
+                      'packages' = @()}
+
+  # Get Googet packages.
+  $out = & 'C:\ProgramData\GooGet\googet.exe' -root 'C:\ProgramData\GooGet' 'installed'
+  $out = $out[1..$out.length]
+  [array]::sort($out)
+
+  foreach ($package_line in $out) {
+    $split = $package_line.Trim() -split '\s+'
+    $name = $split[0]
+    $version = $split[1]
```

Comment:
To avoid the string manipulation, any thoughts on adding a flag to googet that allows the package list to be output in JSON? Perhaps the TODO mentioned below would give you that opportunity.

Reply:
json is just another form of string manipulation :) The key is you just need to make sure the format is static